resolwe_bio/tests/processes/test_clustering.py

```diff
@@ -372,6 +399,33 @@ def test_gene_clustering(self):
                 name='expression 7',
                 source='ENSEMBL',
                 species='Homo sapiens')
+            expression_8 = self.prepare_expression(
+                f_exp='clustering_int1.tab.gz',
+                f_type='TPM',
+                name='expression int 1',
+                source='ENSEMBL',
+                species='Homo sapiens',
+            )
+            expression_9 = self.prepare_expression(
+                f_exp='clustering_int2.tab.gz',
+                f_type='TPM',
+                name='expression int 2',
+                source='ENSEMBL',
+                species='Homo sapiens',
+            )
+
+        inputs = {
+            'exps': [
+                expression_8.pk,
+                expression_9.pk,
+            ],
+            'preprocessing': {
+                'log2': False,
+            },
+        }
+        clustering = self.run_process('clustering-hierarchical-genes', inputs)
+        self.assertEqual(len(clustering.process_warning), 0)
+        saved_json, test_json = self.get_json('clustering_out_genes.json.gz', clustering.output['cluster'])
```

Comment:
Shouldn't you also perform some check of equality between `saved_json` and `test_json`? Same applies to the modified test up.